ui/webui/test/helpers/installer.py

```diff
@@ -26,19 +26,40 @@ class Installer():
     def __init__(self, browser):
         self.browser = browser
 
-    def begin_installation(self):
+    def begin_installation(self, should_fail=False, confirm_erase=True):
+        current_step_id = self.get_current_page_id()
         self.browser.click("button:contains('Begin installation')")
-        self.browser.click("#installation-review-disk-erase-confirm")
 
-    def next(self):
+        if confirm_erase:
+            self.browser.click("#installation-review-disk-erase-confirm")
```

Comment:
Tihs can be parameterized: 
            self.browser.click(f"#{review_id}-disk-erase-confirm")

Reply:
Interesting, so the ids are put together in a similar way when the page is generated?

Reply:
Correct. https://github.com/rhinstaller/anaconda/blob/master/ui/webui/src/components/review/ReviewConfiguration.jsx#L116